### ISSUE TITLE
[css-cascade-3] Editorial: Use logical properties in examples

### ISSUE THREAD

**Issue by r12a** [w3c/csswg-drafts#5570]
4.7. Examples
https://www.w3.org/TR/css-cascade-3/#stages-examples

The table shows numerous example properties, but they are all physical. To encourage use of logical properties, why not convert many of them to logical properties?  eg. text-align value could be start, border-top-width etc could be border-block-start-width etc, width and height could be block-size and inline-size.

The i18n WG believes that this will help raise the profile and subsequent use of logical properties.

**Comment by fantasai** [comment]
Hi @r12a,
The cascading and value processing of logical properties is somewhat complicated and not entirely resolved, and also this spec is trying to transition to REC whereas logical properties is still WD, so I think using logical properties in this example would not be a good idea. (I think it *is* a good idea for examples in our various layout specs, though, where the processing model doesn't matter, only the input and end result.)

As for logical values like `start`, while we have clarity on what the computed value is, it's not entirely clear to me whether the “used value” is left or right, and that question while currently theoretical might end up being non-theoretical in the future, so I'd rather not lock it down one way or another with an example here. :)

So if you don't mind, I'd prefer to reject the suggestion against this particular example in Cascade. Though I'm quite open to using logical properties and values in our examples more generally.
Let me know if you find this acceptable~

**Comment by r12a** [comment]
I guess that the issues you are alluding to are those in the css-logical issue [Order of inheritance vs. mapping in logical properties https://github.com/w3c/csswg-drafts/issues/3029) (?)

If so, shouldn't we have an open issue against the Cascade spec indicating that this needs to be resolved for this spec also?  (It could just point to the css-logical issue for discussion, but i think there needs to be some indication that there's an open issue of that kind for css-cascade.)

**Comment by r12a** [comment]
@fantasai please change the tag on this to css-cascade-4, so that it doesn't cause you transition problems, and doesn't get forgotten for new work. Thanks.

**Comment by fantasai** [comment]
Pushing out to L6 because L4/L5 should be transitioning to CR soon (they're just blocked on some editing details for describing `@import` using the fetch framework or something). This was deferred pending css-logical-1 details solidifying, and that's still not happened.
